docs/advanced-features/athena.md

```diff
@@ -99,18 +446,24 @@ FROM npv
 JOIN
   mv
 ON
-  npv."hash" = mv."tophash" 
+  npv."hash" = mv."tophash"
+"""
 ```
 
-### View of object-Level metadata
+### D. View of object-Level metadata
 The DDL below creates a view that contains package contents, including:
```

Comment:
Please, add `size`. I need `size` to create packages from Athena results. Minimum data to start creating package from entries is `logical_key` to name file entry, `physical_keys` to point where to get file entry, and `size`. Size is just for UI, and in theory can be omitted and re-requested during package creation, but frontend requires it for now.

Reply:
Added to the object_view